src/main/java/org/openlmis/stockmanagement/repository/SourceDestinationAssignmentRepository.java

```diff
@@ -23,9 +23,11 @@
 import org.springframework.data.repository.query.Param;
 
 @NoRepositoryBean
-public interface SourceDestinationAssignmentRepository<T extends SourceDestinationAssignment> 
+public interface SourceDestinationAssignmentRepository<T extends SourceDestinationAssignment>
     extends PagingAndSortingRepository<T, UUID> {
 
+  List<T> findAll();
```

Comment:
do we need this method when we extend PagingAndSortingRepository?

Reply:
Yes, because PagingAndSortingRepository returns Iterable<T> so to avoid unnecessary casting I have added this method.

Reply:
Can we change PagingAndSortingRepository to JpaRepository?

Reply:
That is a great idea. Thank you @lukaslew.